Изменение цен карго.

Patrioy opened the pull request:
## ✨ Обновление каравана и прочие правки

> **Основная суть данного PR — переработка каравана.**
> 
> Караван не будет прежним...  
> _By: @Patrioy_

---

### 🛠 Основные изменения:

- 💰 **Караван получил крупные правки в экономике**:
  - Убраны и сбалансированы способы абуза, связанные с продажей тел, котлет и т.д.
  - Цены на товары были **массово пересмотрены**:
    - Некоторые предметы теперь стоят **дороже**.
    - Другие — **дешевле**.
  - Цель: сделать торговлю более **взвешенной и ролевой**.

- 🧾 **Стартовая сумма крышек** в терминале каравана снижена до **50**:
  - Караван не может скупать топовую снарягу с раундстарта.
  - Требуется активное участие в торговле и RP.

- 🪙 **Ауреи и Динарии теперь имеют цену** в терминале:
  - Ауреи: **100 крышек**.
  - Динарии: **4 крышки**.

- 🛒 В консоль каравана добавлен **новый товар**.

---

### ⚙️ Общие правки и баланс:

- 🧯 **Пулемёты** получили **нёрф**.
- 🧃 В **таблетницу теперь можно складывать только таблетки**.
- 📉 **12.7 калибр** стал **слабее**.
- 🚫 **Перки "Слепота" и "Немота"** теперь **недоступны Рейнджерам НКР** (а также некоторым другим ролям).
- 🛡 **Галеи легионеров** были исправлены:
  - Также увеличено время, нужное для получения **шлемов**.
- 💪 **Броня супермутанта** ослаблена.
- 🧤 **Механик Города** теперь имеет **изолированные перчатки**.
- 📘 **Гайды** получили ряд правок:
  - Добавлен 1 новый (который был, но не был подключён ранее).
- 🌐 **Мелкие исправления в переводах**.

---


Мёржить можно, если больше нихуя добавить не нужно. (Только после вырезания с Сновивейла раундстарт аурей и динариев, ёпты).

Patrioy committed:
Ура, изменение цен!

Изменению подверглись:
Тушки(головы, конечности).
цены за некоторые медикаменты(а то с фулл стаком там пиздец денег даёт).
Цены за браминов.
Цены за товары в консоли заказа грузов.
Цена за полу живого чела(щас бы продавать людей, мммм!!!)

Patrioy committed:
Мелкий нёрф 12.7

С 18 до 16 понизил дмг, чтобы 5.56 хотя-бы немного ощущалась более престижным вариантом.

MrLopLop commented on Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:
```diff
@@ -317,7 +317,7 @@
   - type: Stack
     stackType: AloeCream
-    count: 15 #Was 10, Buffed due to limb damage changes
+    count: 5 #Corvax-Change
```
ЭМЕ КАК Я БУДУ ХИЛИТСЯ НА УБЕЖКЕ?

Patrioy replied:
А ой, не то тронул, АХАХХАХАХА

Patrioy replied:
Всё намана, я парешал!!!!

Patrioy committed:
Фиксы цен

А, э, я случайно вместо цены, стак порезал, бывает и такое.

Вместе с этим уменьшил цены на некоторые доп медикаменты.

Patrioy committed:
Котлеты, уменьшение цены

у котлет есть компонент Material, который отвечает за цену 15, в связи с чем приходится ставить цену -10, дабы не лезть в компонент Material, ибо он используется в других местах.

А по поводу снижения цены - ну, если разрезать одно мясо, получится 3 котлеты каждая из которых стоит как мясо, от чего случается - Увы, момент.

Patrioy committed:
Легионер лодаут фикс + аптейд / Речайндж перков

Пофиксил 1 из шлемов декана, повысил требуемое время (53 минуты заради самого крутого шлема - это пизда).

Убрал возможность немоты рейнджерам, убрал возможность слепоты некоторым ролям(в коде чекнуть можно кому именно).

Patrioy committed:
Доп фикс карго

ещё чучуть цены поднял.

Patrioy committed:
Опа, сюда ещё одно изменение цен.

Patrioy committed:
Правки по гайдам

Незначительные правки по гайдам, добавил отображение основ выживания, слегка подправил пару описаний и добавил инфу по легиону.

Patrioy committed:
Правка брони супермутанта

Понёрфил её, усё.

Patrioy committed:
Фикс повязки

Теперь в меню стрипинга(когда можно шмотку спиздить), повязка не перекрывает наушник(Ура. мы не будем гибать людей заради гарнитуры).

Patrioy committed:
чуть забыл

MrLopLop commented on Resources/Prototypes/InventoryTemplates/human_inventory_template.yml:
```diff
@@ -64,7 +64,7 @@
       slotFlags: BANDAGE
       uiWindowPos: 2,3
-      strippingWindowPos: 2,0
+      strippingWindowPos: 2,1
```
убэри я сам уже фиксанул там

Patrioy replied:
Сам убери у себя, хи-хи ха-ха.

Ладно, ща.

Patrioy replied:
Убрал.

Patrioy committed:
Revert "Фикс повязки"

This reverts commit ae60dc98a114d8089027db2811c7565457c7e88a.

Patrioy committed:
нёрф старт суммы

Дяде патру не нравится, что караванщики раундстартом могут позволить себе купить одну из самых топовых мк2 бронек.
Мы такое осуждаем, и в качестве теста - уменьшаем стартовую сумму, дабы сподвигнуть караван - к торговле с игроками.

Patrioy committed:
смена прайсов + перчатки механику

Patrioy committed:
Update currency.yml

Patrioy committed:
Цены + Топливный Бак в продаже

Ещё один коммит с ещё одним изменением цен на товар.
+ Добавил топливный бак в продажу

Patrioy committed:
Update cargo_armory.yml

Patrioy committed:
Мне лень писать крутой и классный коментарий по тому пру, окей?

Patrioy committed:
очередной коммит с кучей чейнджей.

Patrioy committed:
ок, ща

Patrioy committed:
d

MrLopLop commented on Resources/Prototypes/_Nuclear14/Entities/Objects/Misc/currency.yml:
```diff
@@ -299,6 +299,9 @@
   name: aureus
   description: A gold coin used by the legion and tribes as currency.
   components:
+  - type: PhysicalComposition
```
Файл не наш, в нём тоже нужны пометки.

Patrioy replied:
Ммм, ауреи и динарии оставляла Зекинс, они в файле были.

Patrioy replied:
Там даже вроде от неё пометки были что "Динарии" и "Ауреи"

MrLopLop commented on Resources/Prototypes/_Nuclear14/Guidebooks/nuclear14.yml:
```diff
@@ -5,4 +5,5 @@
   - Factions
-  - WeaponsAmmo
\ No newline at end of file
+  - WeaponsAmmo
```
пометочки

Patrioy replied:
есть

MrLopLop commented on Resources/Prototypes/_Nuclear14/Traits/speech.yml:
```diff
@@ -70,6 +70,7 @@
         - TownSheriff
         - NCRLT
         - NCRRangerVeteran
+        - NCRRanger
```
пометочка

Patrioy replied:
есть

MrLopLop commented on Resources/Prototypes/_Nuclear14/Traits/visual.yml:
```diff
@@ -3,6 +3,22 @@
   category: Visual
   points: 10
   requirements:
+    - !type:CharacterJobRequirement
```
пометка

Patrioy replied:
есть

Patrioy committed:
ок

MrLopLop reviewed (approved):
гуде
